### PR TITLE
fix empty slot in contributors

### PR DESCRIPTION
**Pull request by Mooy1**
## Description
fixed empty slot in contributors

## Changes
added slot 53 to draw background

## Related Issues
idk

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

File: src/main/java/io/github/thebusybiscuit/slimefun4/core/guide/options/ContributorsMenu.java [hunk 1]
```diff
@@ -37,7 +37,7 @@ public static void open(Player p, int page) {
         menu.setEmptySlotsClickable(false);
         menu.addMenuOpeningHandler(pl -> pl.playSound(pl.getLocation(), Sound.BLOCK_NOTE_BLOCK_HARP, 0.7F, 0.7F));
 
-        ChestMenuUtils.drawBackground(menu, 0, 2, 3, 4, 5, 6, 7, 8, 45, 47, 48, 49, 50, 51, 52);
+        ChestMenuUtils.drawBackground(menu, 0, 2, 3, 4, 5, 6, 7, 8, 45, 47, 48, 49, 50, 51, 52, 53);
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/core/guide/options/ContributorsMenu.java:40]: Looks like this was a typo, 52 is the one that shouldn't be here.

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

**Comment by TheBusyBiscuit** [comment]
Also can you please test this first?

**Comment by Mooy1** [comment]
image

**Review by poma123 — APPROVED** [review]
LGTM

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**